src/helpers/chains.ts

```diff
@@ -0,0 +1,107 @@
+import {
```

Comment:
The source of truth now for all supported chains stuff is @tableland/evm... this module simple takes that information, extracts what it needs, and exposes this stuff via helper functions. So it is really easy to determine which validator gateway to target based on a given query, or signer connection. You'll notice that we don't actually define _any_ chain specific state here at all ourselves. It all derives from @tableland/evm.